components/dashboard/src/settings/Preferences.tsx

```diff
@@ -118,50 +110,47 @@ export default function Preferences() {
-                {desktopIdeOptions && desktopIdeOptions.length > 0 && <>
-                    <div className="mt-4 space-x-4 flex">
-                        <CheckBox
-                            title={<div>Open in Desktop IDE <PillLabel type="warn" className="font-semibold mt-2 py-0.5 px-2 self-center">Beta</PillLabel></div>}
-                            desc="Choose whether you would like to open your workspace in a desktop IDE instead."
-                            checked={useDesktopIde}
-                            onChange={(evt) => actuallySetUseDesktopIde(evt.target.checked)} />
+                {browserIdeOptions && <>
+                    <h3>Default Browser Editor</h3>
+                    <p className="text-base text-gray-500 dark:text-gray-400">Choose which IDE you want to use.</p>
+                    <div className="my-4 gap-4 flex flex-wrap">
+                        {
+                            browserIdeOptions.map(([id, option]) => {
+                                const selected = defaultIde === id;
+                                const onSelect = () => actuallySetDefaultIde(id);
+                                return renderIdeOption(option, selected, onSelect);
+                            })
+                        }
                     </div>
-                    {useDesktopIde && <>
-                        <div className="my-4 space-x-4 flex">
-                            {
-                                desktopIdeOptions.map(([id, option]) => {
-                                    const selected = defaultDesktopIde === id;
-                                    const onSelect = () => actuallySetDefaultDesktopIde(id);
-                                    return renderIdeOption(option, selected, onSelect);
-                                })
-                            }
-                        </div>
-
-                        {ideOptions.options[defaultDesktopIde].notes &&
-                            <InfoBox className="my-5 max-w-2xl"><ul>
-                                {ideOptions.options[defaultDesktopIde].notes?.map((x, idx) => <li className={idx > 0 ? "mt-2" : ""}>{x}</li>)}
-                            </ul></InfoBox>
+                    {ideOptions.options[defaultIde] && ideOptions.options[defaultIde].notes &&
+                        <InfoBox className="my-5 max-w-2xl"><ul>
+                            {ideOptions.options[defaultIde].notes?.map((x, idx) => <li className={idx > 0 ? "mt-2" : ""}>{x}</li>)}
+                        </ul></InfoBox>
+                    }
+                </>}
+                {desktopIdeOptions && <>
+                    <h3 className="mt-12">Default Desktop Editor</h3>
+                    <p className="text-base text-gray-500 dark:text-gray-400">Open new workspaces with your preferred choice of desktop editor. Whilst using a desktop editor,<br />you can also work concurrently via the browser editor.</p>
+                    <div className="my-4 gap-4 flex flex-wrap">
```

Comment:
**praise**: Thanks so much for wrapping the options here, @mustard-mh! 🌟

See also :a: [relevant discussion](https://gitpod.slack.com/archives/C02CBKZEVS8/p1638795733242600?thread_ts=1638366112.229500&cid=C02CBKZEVS8) (internal) and :b: https://github.com/gitpod-io/gitpod/pull/7031#discussion_r766124754 for future reference.

Reply:
Another PR later will be good (